docker-hub/release-notes.md

```diff
@@ -11,6 +11,10 @@ known issues for each Docker Hub release.
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+## 2022-09-19
+### Enhancement
+
+In Docker Hub, you can now download a [registry.json](../docker-hub/configure-sign-in.md) file for your organization or copy the commands to create a registry.json file for your organization.
```

Comment:
Maybe worth adding the goal of enforcing sign-in:

Suggestion:
 In Docker Hub, you can now download a [registry.json](../docker-hub/configure-sign-in.md) file or copy the commands to create a registry.json file **to enforce sign-in** for your organization.

Reply:
Thanks, @marisimon. It's been updated in a following commit. PTAL